pkg/state/state_whereabouts.go

```diff
@@ -0,0 +1,129 @@
+/*
```

Comment:
can you modify the file name to be consistent with the state struct name.

`state_whereabouts_cni.go`